token/program-2022/src/extension/confidential_transfer_fee/processor.rs

```diff
@@ -65,22 +60,31 @@ fn process_initialize_confidential_transfer_fee_config(
     extension.authority = *authority;
     extension.withdraw_withheld_authority_elgamal_pubkey =
         *withdraw_withheld_authority_elgamal_pubkey;
+    extension.harvest_to_mint_disabled = false.into();
```

Comment:
So we don't forget to initialize it properly
```suggestion
    extension.harvest_to_mint_enabled = true.into();
```